index.html

```diff
@@ -179,8 +200,10 @@ <h1 class="title">Professional Art <br>Printing Data</h1>
         <div class="about-me">
             <h1> About Me</h1>
             <p>
-                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
-                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                Hello I’m a software developer! I can help you build a
```

Comment:
- [ ] In the readme.md file, please include a `Getting started` section with instructions on how a user can clone and run the project.